proto/authenticator.ridl

```diff
@@ -189,6 +189,19 @@ struct VerificationContext
   - expiresAt: timestamp
 
 
+struct Page
+  - limit?: uint32
+    + go.field.type = uint32
+    + go.tag.json = limit,omitempty
+  - after?: string
+    + go.field.type = string
+    + go.tag.json = after,omitempty
+
+
+enum Migration: string
+  - OIDCToStytch
```

Comment:
for migration names, what do you think if we prefix by a number?

for example update the code to..

```go
enum Migration: string
  - 001_ OIDCToStytch
```

then for the Email migration in the other PR we have `- 002_ OIDCToEmail` .. and we will always ensure migrations run in order. However, maybe its not necessary as they don't necessarily have a dependency on eachother

Reply:
if there is no requirement for order of migrations, then there probably isn't any value to this as with typical migrations like in RDMBS'

as for starting with a number, if that is problem, could always prefix with a letter like, "M001_XXX" .. but doesn't sound like we need this. The idea of migration numbers is that you record which migrations have been run, so they don't get triggered again once they are run, which, we may want to do in fact..

Reply:
In the case of authenticator migrations, each of them is idempotent so there's no risk in triggering them multiple times